Lib/nntplib.py

```diff
@@ -96,7 +96,7 @@
 class NNTPError(Exception):
     """Base class for all nntplib exceptions"""
     def __init__(self, *args):
-        Exception.__init__(self, *args)
+        super().__init__(*args)
         try:
```

Comment:
This change seems unrelated. Is it required?

Reply:
@ericvsmith Yes this change is unrelated, should I create for the separate PR?

Reply:
I'd put it in a separate PR. But is this a common pattern in the stdlib? I looked through a few libraries and couldn't find any instances of an exception that takes any parameters, so I'm not sure what best practice is here. I guess it can't hurt.

Reply:
I observed this kind of simple refactoring works was done by other huge work.

Reply:
My concern here is not so much including this in this PR (I'd move it to another PR, but I won't object to leaving it here).

My concern is: is an exception calling `super()`, such that it can be used with cooperative multiple inheritance, a common thing in the stdlib? Or is this the only place in the stdlib that it's done? If this is the only place it will be used in the stdlib, we should either change every usage in the the stdlib to use `super()`, or not make this change at all.